### PR TITLE
Add notes in tutorials

### PR DESCRIPTION
**Pull request by will-jl944**

### CODE REVIEW

File: tutorials/slim/prune/object_detection/yolov3_train.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import paddlex as pdx
 from paddlex import transforms
```
Comment (FlyingQianMM) [inline tutorials/slim/prune/object_detection/yolov3_train.py]: as T

File: tutorials/slim/prune/semantic_segmentation/unet_prune.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import paddlex as pdx
 from paddlex import transforms
```
Comment (FlyingQianMM) [inline tutorials/slim/prune/semantic_segmentation/unet_prune.py]: as T

File: tutorials/slim/prune/semantic_segmentation/unet_train.py [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import paddlex as pdx
 from paddlex import transforms
```
Comment (FlyingQianMM) [inline tutorials/slim/prune/semantic_segmentation/unet_train.py]: as T

File: tutorials/slim/prune/object_detection/yolov3_train.py (hunk 1 above)
Comment (will-jl944) [reply tutorials/slim/prune/object_detection/yolov3_train.py]: Done

File: tutorials/slim/prune/semantic_segmentation/unet_prune.py (hunk 2 above)
Comment (will-jl944) [reply tutorials/slim/prune/semantic_segmentation/unet_prune.py]: Done

File: tutorials/slim/prune/semantic_segmentation/unet_train.py (hunk 3 above)
Comment (will-jl944) [reply tutorials/slim/prune/semantic_segmentation/unet_train.py]: Done

**Pull request merged**